packages/react-native/Libraries/LogBox/Data/LogBoxData.js

```diff
@@ -198,16 +201,17 @@ export function addLog(log: LogData): void {
   // otherwise spammy logs would pause rendering.
   setImmediate(() => {
     try {
-      const stack = parseErrorStack(errorForStackTrace?.stack);
+      const defaultStack = parseErrorStack(errorForStackTrace?.stack);
```

Comment:
Try this to reuse the same parsing logic for both

```suggestion
      const stack = parseErrorStack(log.stack ?? errorForStackTrace.stack);
```